src/Model/CategoryManagerInterface.php

```diff
@@ -20,7 +20,7 @@
 /**
  * @method PagerInterface         getRootCategoriesPager(int $page = 1, int $limit = 25, array $criteria = [])
  * @method PagerInterface         getSubCategoriesPager(int $categoryId, int $page = 1, int $limit = 25, array $criteria = [])
- * @method CategoryInterface[]    getRootCategoriesForContext(ContextInterface|string|null $context)
+ * @method CategoryInterface[]    getRootCategoriesForContext(ContextInterface|null $context)
```

Comment:
This was wrong: https://github.com/sonata-project/SonataClassificationBundle/blob/3.x/src/Entity/CategoryManager.php#L122